pennylane/_qubit_device.py

```diff
@@ -177,13 +177,13 @@ def _const_mul(constant, array):
 
     **Example:**
 
-    Let's create device that inherits from :class:`~pennylane.devices.DefaultQubit` and overrides the
+    Let's create device that inherits from :class:`~pennylane.devices.DefaultQubitLegacy` and overrides the
```

Comment:
I wonder if we should now put a note in `QubitDevice` and `Device` to suggest use of the new device API (doesn't need to be in this PR).

Reply:
sure! who exactly is the note for, people trying to make their own device and choosing to inherit from the old API, so we nudge them to the new one? Or just for people using it?

Reply:
The former, i.e., device/plugin developers.